README.md

```diff
@@ -57,7 +48,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: this is O(n^2) because there are two nested .each loops.
```

Comment:
Sorry for the delay! Is it because the words and ending arrays aren't dependent on each other? For some reason I thought (n*m) converted to (n**2) but maybe that's incorrect!